mod/researched_metrics/set/right/checked_by.rb

```diff
@@ -106,21 +106,10 @@ def option_label_text _option_name
   end
 
   view :core, template: :haml
-
-  # FIXME: this view is wrongly cached if it's moved to a haml template
-  #   To see how it fails add `template: :haml` and run the double_check.feature
```

Comment:
one FIXME for me ;)

I think my view handling updates cleaned this up. Seems to work fine now.